robottelo/decorators/__init__.py

```diff
@@ -199,6 +199,8 @@ class ProjectModeError(Exception):
     """Indicates an error occurred while skipping based on Project Mode."""
 
 
+# This defination is not being used currently as there are no multiple projects to run the tests on
+# Also, The setting project has been removed from properties untill wnd if we need in feature
 def run_only_on(project):
```

Comment:
What is point in keeping this definition if we are removing test class definition as well https://github.com/SatelliteQE/robottelo/pull/6976/files#diff-86bbbc36dd5587c78264c1b8f556a6b7L930

Reply:
As I said in comments above that this can be used later if a similar sort of scenario comes where we need to choose between two things.

About the test cases, if you saw those tests, they were tightly coupled with 'project' property in robottelo.propoerties which I have removed in this PR. Hence whenever you run 'make test', the tests will fail with no property found even though we have a definition in place.

So let's keep the definition hanging as it's not being used anywhere right now but can be used later. We can write tests later as per the function use.